install-deps.sh

```diff
@@ -407,7 +407,7 @@ EOF
                 ;;
             rocky|centos|rhel|ol|virtuozzo)
                 MAJOR_VERSION="$(echo $VERSION_ID | cut -d. -f1)"
-                $SUDO dnf install -y dnf-utils
+                $SUDO dnf install -y dnf-utils selinux-policy-targeted
```

Comment:
What about adding this to the ceph.spec BuildRequires instead? IMHO build dependencies should be stated in a single place.

Reply:
i've added it in ceph.spec, i think i added it in the right place

Reply:
I'm actually wondering if we should move this back to `install-deps.sh`. Technically all we need to build the RPM inside a mock chroot is `selinux-policy-devel`, not `selinux-policy-targeted`

Reply:
done! @ktdreyer